packages/api/src/controllers/farmController.js

```diff
@@ -188,32 +200,77 @@ const farmController = {
           error: e.message ? e.message : e,
         });
       }
-    }
+    };
   },
 
   async getUser(req, trx) {
     // check if a user is making this call
     if (req.user) {
-
       const uid = req.user.user_id;
 
       return await userModel.query(trx).where(userModel.idColumn, uid).returning('*');
     }
   },
 
   async insertUserFarm(user, farm_id, trx) {
-    return userFarmModel.query(trx).insert({
-      user_id: user.user_id,
-      farm_id,
-      role_id: 1,
-      status: 'Active',
-    }).returning('*');
+    return userFarmModel
+      .query(trx)
+      .insert({
+        user_id: user.user_id,
+        farm_id,
+        role_id: 1,
+        status: 'Active',
+      })
+      .returning('*');
   },
 
   async getCountry(country) {
-    const { iso, unit, id } = await knex('countries').select('*').where('country_name', country).first();
+    const { iso, unit, id } = await knex('countries')
+      .select('*')
+      .where('country_name', country)
+      .first();
     return { currency: iso, measurement: unit.toLowerCase(), id };
   },
-}
+
+  async getUTCOffsetFromGridPoints(gridPoints) {
```

Comment:
The update function is not updating the utc_offset on calling updateFarm function in the farmControlller. You need to call the 'const utc_offset = await this.getUTCOffsetFromGridPoints(req.body.grid_points);' to update the value of utc_offset value in the database.